setup.cfg

```diff
@@ -23,7 +23,8 @@ install_requires =
 
     # pytorch backend
     torch;python_version>='3.7'
-    sbi<=0.22.0;python_version>='3.7'
+    torch_geometric;python_version>='3.7'
```

Comment:
torch_geometric is not strictly necessary unless you want to do graph embedding networks. I think it also conflicts with some cuda environments, so best to leave it out